### ISSUE TITLE
BO - login page - long shop name overlap the Back link below

### ISSUE THREAD

**Issue by RicRey1988** [PrestaShop/PrestaShop#27800]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

after upgrading from prestashop 1.7.7.7 to 1.7.8.3 in the administration area login is seen as unbalanced, I have already cleared the cache...
![Screenshot_306](https://user-images.githubusercontent.com/44790517/155856687-00629a94-5bb5-4821-8d87-c0af06d0af65.png)

### Steps to reproduce

1) upgrade from prestashop 1.7.7.7 to 1.7.8.3
2) verify BO login

### PrestaShop version(s) where the bug happened

1.7.8.3

### PHP version(s) where the bug happened

7.3

**Comment by hibatallahAouadni** [comment]
Hello @RicRey1988 

Thanks for reporting the issue!
I reproduce the issue with PS1783, PS1780 and PS1778, see the attached screen record below:

https://user-images.githubusercontent.com/23308427/155967775-afd5b6b1-6dc4-4f1f-a99f-74169938dd8c.mp4

With PS1778, the title must be 6 lines to reproduce the issue and that's kind of absurd to have a title that long.
While with PS1780, we encounter the issue just with 2 lines long title. So, it's a **regression**!
So, I propose to move this tag:
```
<p class="text-center">
   <i class="icon-caret-left"></i> Back to <a href="http://prestashop/prestashop_1.7.8.0/">presta1780</a>
</p>
```
Under the login-footer div to resolve the issue.
Ping @NeOMakinG wdyt of the proposed solution?

PS: it's not related to upgrade PS, you can encounter the issue even with a fresh install!

Thanks!

**Comment by Hlavtox** [comment]
Who the hell puts fixed heights on things!!! 🤦‍♂️

**Comment by RicRey1988** [comment]
Actually, it doesn't seem absurd to me, since in the general title of the store it is specified a little what the store is, the strange thing is that they have put it as fixed...

The solution seems simple, but they should leave it for future versions, it's something that doesn't affect anyone and benefits some of us who use big titles...

I have been using Prestashop since its version 1.3 and my store has always been updated. I thought that it was one of the problems of the update because it is the first time that it happens to me with the BO login...

**Comment by NeOMakinG** [comment]
@hibatallahAouadni the fixed height should be removed, it can be a min-height for example, but not a fixed height! Getting the text out is not the right solution to me

**Comment by NeOMakinG** [comment]
Fixed on 1.7.8.x branch
